test/unit/StubbedStreamrClient.ts

```diff
@@ -1,19 +1,23 @@
-import { StreamrClient } from '../../src/'
+import { StreamrClient } from '../../src/index'
 import { Stream } from '../../src/stream'
 
 export default class StubbedStreamrClient extends StreamrClient {
+    // @ts-expect-error
+    // eslint-disable-next-line class-methods-use-this
     getUserInfo() {
         return Promise.resolve({
             username: 'username',
         })
     }
 
-    async getStream () {
-        return new Stream(null, {
+    // eslint-disable-next-line class-methods-use-this
+    async getStream() {
+        return new Stream(null as any, {
```

Comment:
this seems weird

Reply:
Did a small change to the implementation and now there is less casting and disabled warnings.

Reply:
excellent